### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by alex-ju**
Create a minimal [README.md](https://github.com/kevindew/paste-html-to-govspeak/blob/e3fdbc7e90fe611c3477ea5372d291b90ea51330/README.md) explaining what the script does and how can be used 📝

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+To continuously build files while developing run:
+
+```
+npm run watch
+```
+
+## License
+
+Distributed under the MIT license. See LICENSE for details.
```
Comment (kevindew) [inline README.md]: I think you'll need to drop a LICENSE file in too. Can just copy this and change the year: https://github.com/alphagov/content-publisher/blob/master/LICENCE

Comment (alex-ju) [reply README.md]: Yep, added.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Browser support
+
+- Chrome
+- Firefox
+- Safari
+- Internet Explorer 11
+- Microsoft Edge
+
+[govspeak]: https://github.com/alphagov/govspeak
```
Comment (kevindew) [inline README.md]: Sorry, minor but this link seems to have ended up in the middle of this file.

Comment (alex-ju) [reply README.md]: Good spot. Moved at the end of the file.

**Pull request merged**